codegen/src/test/resources/software/amazon/awssdk/codegen/poet/transform/eventonemarshaller.java

```diff
@@ -1,43 +0,0 @@
-package software.amazon.awssdk.services.jsonprotocoltests.transform;
-
-import software.amazon.awssdk.annotations.Generated;
-import software.amazon.awssdk.annotations.SdkInternalApi;
-import software.amazon.awssdk.core.exception.SdkClientException;
-import software.amazon.awssdk.core.protocol.MarshallLocation;
-import software.amazon.awssdk.core.protocol.MarshallingInfo;
-import software.amazon.awssdk.core.protocol.MarshallingType;
-import software.amazon.awssdk.core.protocol.ProtocolMarshaller;
-import software.amazon.awssdk.services.jsonprotocoltests.model.EventOne;
-import software.amazon.awssdk.utils.Validate;
-
-/**
- * {@link EventOne} Marshaller
- */
-@Generated("software.amazon.awssdk:codegen")
-@SdkInternalApi
-public class EventOneMarshaller {
```

Comment:
Should not be deleted?

Reply:
I just saw they were using classes that didn't exist anymore so figured they weren't used and removed them

Reply:
I can add them when I fix the event marshaller generation